tests/adapters/anthropic/v1/messages/anthropic.test.ts

```diff
@@ -0,0 +1,215 @@
+import { describe, it, expect, beforeEach, vi } from "vitest";
+import { AnthropicMessagesAnthropicAdapter } from "../../../../../src/adapters/anthropic/v1/messages/anthropic";
+
+describe("messages/adapter smoke", () => {
+  it("runs", () => {
+    expect(true).toBe(true);
+  });
+});
+
+// ---- Mock @anthropic-ai/sdk ----
+let createImpl: (...args: any[]) => any;
+
+vi.mock("@anthropic-ai/sdk", () => {
+  class FakeAnthropic {
+    static lastOpts: any;
+    messages = {
+      create: (...args: any[]) => createImpl(...args),
+    };
+    constructor(opts: any) {
+      FakeAnthropic.lastOpts = opts;
+    }
+  }
+  return { default: FakeAnthropic };
+});
```

Comment:
I think it's better to study how anthropic tests their code.

https://github.com/anthropics/anthropic-sdk-typescript/blob/main/tests/responses.test.ts

They overwrite the fetch function